config/locales/diaspora/en.yml

```diff
@@ -951,7 +951,7 @@ en:
     delete_link: "Delete item"
     reported_user_details: "Details on reported user"
     confirm_deletion: "Are you sure to delete the item?"
-    not_found: "<u>The post/comment was not found. It seems that it was deleted by the user!</u>"
+    not_found: "The post/comment was not found. It seems that it was deleted by the user!"
```

Comment:
I removed the underlining here because that makes it harder to distinguish it from a link.